Makefile

```diff
@@ -0,0 +1,353 @@
+RUST_LOG ?= debug
+# you can use devnet | testnet | mainnet
+chain ?= devnet
+# set the data path when the claster run
+data_path ?= /tmp/data
+# you can add a prefix such as node0 | node1 to create mulpile clusters on one host
+prefix ?=
+sgx_mode ?= HW
+# rebuild
+rebuild ?=
+ifeq ($(rebuild)x, x)
+	CARGO_BUILD_CMD = cargo build --release
+	MAKE_CMD := make
+else
+	CARGO_BUILD_CMD = cargo clean && cargo build --release
+	MAKE_CMD = make clean && make
+endif
+
+base_port ?= 26650
+TX_QUERY_PORT       = $(shell expr $(base_port) + 1)
+TENDERMINT_P2P_PORT = $(shell expr $(base_port) + 6)
+TENDERMINT_RPC_PORT = $(shell expr $(base_port) + 7)
+CLIENT_RPC_PORT     = $(shell expr $(base_port) + 9)
+
+# the chain version, such as v0.1.0, v0.2.0.
+tag ?=
+# if the tag not set, it will use current tag, if current code is not checkouted to a tag, it will use `develop`
+ifeq ($(tag)x, x)
+	TAG = $(shell git describe --exact-match --tags $(git log -n1 --pretty='%h') 2>/dev/null || echo develop)
+else
+	TAG = $(tag)
+endif
```

Comment:
if the tag is set, it shouldn't rebuild stuff, just fetch the released binaries, e.g. https://github.com/crypto-com/chain/releases/download/v0.2.0/crypto-com-chain-release-0.2.0.tar.gz

Reply:
fixed